runner_manager/jobs/create.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from runner_manager import RunnerGroup
+from runner_manager.clients.github import GitHub
+from runner_manager.dependencies import get_github
+
+log = logging.getLogger(__name__)
+
+
+def runner(group: RunnerGroup) -> str | None:
+    """Job to create a runner within a group.
+
+    Args:
+        RunnerGroup: Runner group to create a runner in
+    """
+    github: GitHub = get_github()
+    runner = group.create_runner(github)
+    if runner is not None:
+        return runner.pk
```

Comment:
if runer is None, it means that the program couldnt create the runner ? What do we do in this case ?

Reply:
Correct, each runner group has a max number of runner allowed, a case where runner is None would be that we asked a runner to be created but we reached `max`.

So in this case we do nothing, as you asked the runner to be created it has been counted in the queue and will be created later on by the program. It's all good here.